### PR TITLE
Fix missing links in JMS charter

### PR DESCRIPTION
**Pull request by andrii-i**

### CODE REVIEW

File: charters/MediaStrategyCharter.md
Comment (jasongrout) [inline charters/MediaStrategyCharter.md:5]: Looks like we are missing a space
```suggestion
The mission of the Jupyter Media Strategy Working Group (JMS) is to ensure that communications in Jupyter official channels are strategic and benefit Project Jupyter. The Jupyter [governance model](https://jupyter.org/governance/standing_committees_and_working_groups.html) includes a distributed model of responsibilities. This means that by design Jupyter publications, social media posts, promotions and other media activity are meant to be community driven. The JMS is vendor neutral and will provide mechanisms to enable the community to speak publicly through the official Jupyter communication channels. The JMS will also help to improve Jupyter media activity by creating strategy and guidelines, serving as editors for existing public media channels, and overseeing creation/delegation of new media channels.
```

**Review by jasongrout — APPROVED** [review]
LGTM, with one minor typo correction

File: charters/MediaStrategyCharter.md
Comment (andrii-i) [reply charters/MediaStrategyCharter.md]: Good catch, thank you.

**Review by jake-stack — APPROVED** [review]

**Review by Ruv7 — APPROVED** [review]

**Review by blink1073 — APPROVED** [review]

**Comment by blink1073** [comment]
Merging, since all of us have approved.

**Comment by andrii-i** [comment]
@jupyter/executive-council could you please this PR a look? This PR adds missing links and was merged without explicit approval of EC. Are these changes OK or should it be reverted (see PR #232)?

**Comment by jasongrout** [comment]
We discussed this in a recent meeting. Eventually we should set permissions on this repo so that only the EC and SSC can merge, but for situations like this:

1. The voting members of the working group or standing committee must approve the change (done here)
2. For small or unsubstantial changes to a working group/standing committee charter, any member of the EC can merge. For larger changes then they should notify the EC and ask for review. I think this falls under the "small changes" case. While Steve did the actual merging, Ana and I did approve the change here, so I think we are good.

**Comment by andrii-i** [comment]
Thank you @jasongrout.

**Comment by Ruv7** [comment]
I think we're set on this particular change from the JMS. Thanks all for the work here. 

Since this request will set precedent moving forward I do want to clarify that regardless of the merge rights on this repo, the intent here is that Working Group and Standing Committees consult with the EC for changes to their charters. If an EC member is also a member of the WG/SC they should not be the one to merge the change.

I want to take out the description large and small changes because even small changes - for example increasing a number from 1 to 2 or making a word plural instead of singular can be a substantive when it comes to a charter doc.

Process for changing charter

1. The voting members of the working group or standing committee approve the change 

2. Notify the EC and request review. For changes that are not substantive, any member of the EC can merge. For changes that land in a grey area or are deemed substantive, the EC may request further info or discussion from the group.

**Comment by jasongrout** [comment]
Also, let's write this process down somewhere on the EC team compass.

**Comment by Ruv7** [comment]
> Also, let's write this process down somewhere on the EC team compass.

Agree, this should be documented. I'm not sure where it should go, perhaps in the Governance Docs? I think people who are looking for how to's are more likely to go there than the EC Team Compass. I'm can own the next steps on figuring that out.

**Pull request merged**
